lib/foreman_ansible_core/runner/ansible_runner.rb

```diff
@@ -118,6 +120,10 @@ def verbose?
         @verbosity_level.to_i.positive?
       end
 
+      def forks
+        '--forks ' + (@ansible_forks.to_i.positive? ? @ansible_forks.to_s : '5')
```

Comment:
I'd say we should not enforce a default value here. If an override is not provided by the user, let's let ansible decide on its own

Reply:
Agreed @adamruzicka please take another look when you can.